app/src/main/java/org/oppia/app/help/faq/FAQActivity.kt

```diff
@@ -25,4 +31,9 @@ class FAQActivity : InjectableAppCompatActivity() {
       return intent
     }
   }
+
+  override fun onSupportNavigateUp(): Boolean {
```

Comment:
Instead of this, checkout the implementation of back button in `StoryActivity`.

Reply:
@rt4914 Can you tell me a specific position where should I look for? I searched but didn't found anything related to the back button.

Reply:
done